docker/install.sls

```diff
@@ -55,6 +55,8 @@ docker-config:
     - template: jinja
     - mode: 644
     - user: root
+    - context:
+      config: {{ docker.config | json }}
```

Comment:
The indentation is significant, as pointed out by @daks earlier (I'll see if I can find the reference):

```suggestion
        config: {{ docker.config | json }}
```

Reply:
@noelmcloughlin Here you go: https://github.com/orgs/saltstack-formulas/projects/1#card-20142638.